### PR TITLE
sync zh-trans /docs/concepts/workloads/controllers/cron-jobs.md and /…

### PR DESCRIPTION
**Pull request by tanjunchen**
sync content/zh/docs/concepts/workloads/controllers/cron-jobs.md  and content/zh/docs/concepts/workloads/controllers/daemonset.md

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -139,53 +133,56 @@ defaulting was not compatible with `kubectl apply`. Also, once a DaemonSet is cr
...
 
-此外，您通常不应该创建任何 pods，它们的 label 与 selector 匹配，或者直接创建，或者通过另一个 DaemonSet、或者其他控制器，比如 ReplicaSet。否则，DaemonSet 控制器会认为这些 Pod 是由它创建的。Kubernetes 不会阻止你这样做。 您可能希望这样做的一种情况是在节点上手动创建具有不同值的 Pod 以进行测试。
+<!--
+Also you should not normally create any Pods whose labels match this selector, either directly, via
+another DaemonSet, or via another workload resource such as ReplicaSet.  Otherwise, the DaemonSet
+{{< glossary_tooltip term_id="controller" >}} will think that those Pods were created by it.
+Kubernetes will not stop you from doing this. One case where you might want to do this is manually
+create a Pod with a different value on a node for testing.
+-->
+另外，通常不应直接通过另一个 DaemonSet 或另一个工作负载资源（例如 ReplicaSet）来创建其标签与该选择器匹配的任何 Pod。否则，DaemonSet {{< glossary_tooltip term_id="controller" >}}会认为这些 Pod 是由它创建的。Kubernetes 不会阻止你这样做。您可能要执行此操作的一种情况是，手动在节点上创建具有不同值的 Pod 进行测试。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
另外，通常不应直接通过另一个 DaemonSet 或另一个工作负载资源（例如 ReplicaSet）来创建其标签与该选择器匹配的任何 Pod。否则，DaemonSet {{< glossary_tooltip term_text="控制器" term_id="controller" >}}会认为这些 Pod 是由它创建的。Kubernetes 不会阻止你这样做。您可能要执行此操作的一种情况是，手动在节点上创建具有不同值的 Pod 进行测试。
```

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 2]
```diff
@@ -213,10 +209,7 @@ That introduces the following issues:
    is handled by default scheduler. When preemption is enabled, the DaemonSet controller
    will make scheduling decisions without considering pod priority and preemption.
 -->
-
-{{< feature-state state="stable" for-kubernetes-version="1.17" >}}
-
-DaemonSet 确保所有符合条件的节点都运行一个 Pod 的副本。通常，运行 Pod 的节点由 Kubernetes scheduler 选择。然而，DaemonSet pods 由 DaemonSet controller 创建和调度。这将引入以下问题：
+DaemonSet 确保所有符合条件的节点都运行一个 Pod 的副本。通常，运行 Pod 的节点由 Kubernetes scheduler 抉择。然而，DaemonSet pods 由 DaemonSet controller 创建和调度。这将引入以下问题：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
DaemonSet 确保所有符合条件的节点都运行该 Pod 的一个副本。通常，运行 Pod 的节点由 Kubernetes 调度器抉择。不过，DaemonSet pods 由 DaemonSet 控制器创建和调度。这将引入以下问题：
```

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 3]
```diff
@@ -230,7 +223,7 @@ the DaemonSet pod already exists, it is replaced. The DaemonSet controller only
 performs these operations when creating or modifying DaemonSet pods, and no
 changes are made to the `spec.template` of the DaemonSet.
 -->
-`ScheduleDaemonSetPods` 允许您使用默认 scheduler 而不是 DaemonSet 控制器来调度 DaemonSets，方法是将 `NodeAffinity` 添加到 DaemonSet pods，而不是 `.spec.nodeName`。 然后使用默认 scheduler 将 pod 绑定到目标主机。 如果 DaemonSet pod的亲和节点已存在，则替换它。 DaemonSet 控制器仅在创建或修改 DaemonSet pods 时执行这些操作，并且不对 DaemonSet的 `spec.template` 进行任何更改。
+`ScheduleDaemonSetPods` 允许您使用默认 scheduler 而不是 DaemonSet 控制器来调度 DaemonSets，方法是将 `NodeAffinity` 添加到 DaemonSet pods，而不是 `.spec.nodeName`。 然后使用默认 scheduler 将 pod 绑定到目标主机。 如果 DaemonSet pod 的亲和节点已存在，则替换它。 DaemonSet 控制器仅在创建或修改 DaemonSet pods 时执行这些操作，并且不对 DaemonSet 的 `spec.template` 进行任何更改。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
`ScheduleDaemonSetPods` 允许您使用默认调度器而不是 DaemonSet 控制器来调度 DaemonSets，方法是将 `NodeAffinity` 添加到 DaemonSet pods，而不是 `.spec.nodeName`。 然后使用默认调度器将 pod 绑定到目标主机。 如果 DaemonSet pod 的亲和节点已存在，则替换它。 DaemonSet 控制器仅在创建或修改 DaemonSet pods 时执行这些操作，并且不对 DaemonSet 的 `spec.template` 进行任何更改。
```

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 4]
```diff
@@ -248,7 +241,7 @@ In addition, `node.kubernetes.io/unschedulable:NoSchedule` toleration is added
 automatically to DaemonSet Pods. The default scheduler ignores
 `unschedulable` Nodes when scheduling DaemonSet Pods.
 -->
-此外，`node.kubernetes.io/unschedulable：NoSchedule` toleration 会自动添加到 DaemonSet Pods。 在调度DaemonSet Pod 时，默认调度器会忽略 `unschedulable`节点。
+此外，`node.kubernetes.io/unschedulable：NoSchedule` toleration 会自动添加到 DaemonSet Pods。 在调度 DaemonSet Pod 时，默认调度器会忽略 `unschedulable` 节点。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
此外，系统会自动添加 `node.kubernetes.io/unschedulable：NoSchedule` 容忍度到 DaemonSet Pods。 在调度 DaemonSet Pod 时，默认调度器会忽略 `unschedulable` 节点。
```

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 5]
```diff
@@ -260,7 +253,7 @@ the related features.
 -->
 ### Taints and Tolerations
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]: 污点和容忍度

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 6]
```diff
@@ -260,7 +253,7 @@ the related features.
...
 
-尽管 Daemon Pods 尊重[taints and tolerations](/docs/concepts/configuration/taint-and-toleration)，根据相关特性，会自动将以下 tolerations 添加到 DaemonSet Pods 中。
+尽管 Daemon Pods 遵循 [taints 和 tolerations](/docs/concepts/configuration/taint-and-toleration)，根据相关特性，会自动将以下 tolerations 添加到 DaemonSet Pods 中。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
尽管 Daemon Pods 遵循 [污点和容忍度](/docs/concepts/configuration/taint-and-toleration) 规则，根据相关特性，会自动将以下容忍度添加到 DaemonSet Pods 中。
```

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 7]
```diff
@@ -260,7 +253,7 @@ the related features.
...
 
 | Toleration Key                           | Effect     | Version | Description                                                  |
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]: 标题翻一下

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 8]
```diff
@@ -287,49 +283,52 @@ Some possible patterns for communicating with Pods in a DaemonSet are:
 - **Service**: Create a service with the same Pod selector, and use the service to reach a
   daemon on a random node. (No way to reach specific node.)
 -->
-## 与 Daemon Pods 通信
+与 DaemonSet 中的 Pod 进行通信，几种可能的通信模式如下：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
与 DaemonSet 中的 Pod 进行通信的几种可能模式如下：
```

File: content/zh/docs/concepts/workloads/controllers/daemonset.md [hunk 9]
```diff
@@ -287,49 +283,52 @@ Some possible patterns for communicating with Pods in a DaemonSet are:
...
 
+<!--
 If node labels are changed, the DaemonSet will promptly add Pods to newly matching nodes and delete
 Pods from newly not-matching nodes.
 
 You can modify the Pods that a DaemonSet creates.  However, Pods do not allow all
 fields to be updated.  Also, the DaemonSet controller will use the original template the next
 time a node (even with the same name) is created.
+-->
+如果修改了节点标签（Label），DaemonSet 将立刻向新匹配上的节点添加 Pod，同时删除不能够匹配的节点上的 Pod。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/daemonset.md]:
```suggestion
如果修改了节点标签，DaemonSet 将立刻向新匹配上的节点添加 Pod，同时删除不能够匹配的节点上的 Pod。
```

**Comment by tanjunchen** [comment]
/cc @tengqm 
done

**Pull request merged**